subprojects/samples/src/integTest/groovy/org/gradle/integtests/samples/dependencymanagement/SamplesDefiningUsingConfigurationsIntegrationTest.groovy

```diff
@@ -34,7 +33,6 @@ class SamplesDefiningUsingConfigurationsIntegrationTest extends AbstractIntegrat
     }
 
     @UsesSample("dependencyManagement/definingUsingConfigurations-custom")
-    @ToBeFixedForConfigurationCache(iterationMatchers = ".*kotlin dsl")
```

Comment:
I found this test and decided that this change is important because test was failing when launching from Intellij. Let me know if it's wrong and I should not touch it.

Reply:
Indeed, and good catch!